tssc/step_implementers/push_container_image/skopeo.py

```diff
@@ -47,7 +47,9 @@
 REQUIRED_CONFIG_KEYS = [
     'destination',
     'src-tls-verify',
-    'dest-tls-verify'
+    'dest-tls-verify',
+    'service-name',
+    'application-name'
```

Comment:
@kfrankli Should the 'Step Configuration' definition (above) be updated to include 'service-name' and 'application-name'?

Reply:
We aren't doing that for other implementors as it's a global config. Not sure how we want to cover that.

Reply:
After thought, maybe we should make this a bug because I think we should document this requirement in each step implementer that requires the service-name and application-name (a lot). Thoughts @HunterGerlach @itewk ?

Reply:
I agree.

Reply:
Done: https://projects.engineering.redhat.com/browse/NAPSSPO-602